test/integration/submission/migration_utils/schema_builder.ts

```diff
@@ -0,0 +1,142 @@
+import { migrationDiffs } from './stub-diffs';
+import _ from 'lodash';
+import fs from 'fs';
+import { schemaV1_accessible, schemaV2, schemaV3 } from './static-schema-data';
+const DICTIONARY_NAME = 'ARGO Clinical Submission';
+
+export const buildDynamicStubSchema = () => {
+  const convertToDictionaryNotation = (schemaVersion: Object) => {
+    return Object.values(schemaVersion).map(schema => {
+      const { fields, ...rest } = schema;
+      const newFields = Object.values(schema.fields);
+      return _.assign(rest, { fields: newFields });
+    });
+  };
+
+  const legacyDictionaries = [
```

Comment:
It sounds to me like these are the same stubs in the schema-stub file, if so why not just import that??

Reply:
yeah that make sense. I think I had initially planned to do something with these and then decided not to, then got stuck in tunnel vision. thanks!